docs/agents/influxdb_publisher_v2.rst

```diff
@@ -0,0 +1,152 @@
+
+========================
+InfluxDB Publisher v2 Agent
+========================
+
+The InfluxDB Publisher v2 Agent is an upgraded InfluxDB Publisher Agent that
+uses the influxdb-client package instead of influxdb package to work with
+InfluxDB v2.
+
+.. argparse::
+   :module: ocs.agents.influxdb_publisher_v2.agent
+   :func: make_parser
+   :prog: agent.py
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container. Also included is an example for setting up
+Grafana to display data from InfluxDB.
+
+OCS Site Config
+```````````````
+
+Add an InfluxDBAgent to your OCS configuration file::
+
+      {'agent-class': 'InfluxDBAgentv2',
+       'instance-id': 'influxagent',
+       'arguments': [['--initial-state', 'record'],
+                     ['--org', 'socs'],
+                     ['--database', 'ocs_feeds']]},
+
+Docker Compose
+``````````````
+
+Add the InfluxDB Publisher v2 Agent container to your docker-compose file::
+
+  ocs-influxdb-publisher-v2:
+    image: simonsobs/ocs:latest
+    hostname: ocs-docker
+    environment:
+      - INSTANCE_ID=influxagent
+    volumes:
+      - ${OCS_CONFIG_DIR}:/config:ro
+    env_file:
+      - .env
+
+Your .env file should contain credentials for the InfluxDB v2 database::
+
+  INFLUXDB_V2_URL=http://influxdb:8086
+  INFLUXDB_V2_ORG=socs
+  INFLUXDB_V2_TOKEN=<your-token>
+
+You will also need an instance of InfluxDB v2 running somewhere on your network.
+This likely should go in a separate docker-compose file so that it remains
+online at all times. An example compose file would look like::
+
+  version: '3.7'
+  services:
+    influxdb:
+      image: "influxdb:2.7"
+      container_name: "influxdb"
+      restart: always
+      ports:
+        - "8086:8086"
+      environment:
+        - INFLUXDB_HTTP_LOG_ENABLED=false
+      volumes:
+        - /srv/influxdb2:/var/lib/influxdb2
+
+  networks:
+    default:
+      external:
+        name: ocs-net
+
+.. note::
+    This separate docker-compose file setup depends on having a docker network
+    that connects your various docker-compose files. On a single-node setup
+    this can be accomplished with the network settings above in each docker-compose
+    file.
+
+    You then need to create the docker network with::
+
+       $ docker network create --driver bridge ocs-net
+
+    Containers on the network should then be able to communicate.
+
+For more information about configuring Docker Compose files, see the `Compose
+file reference`_.
+
+.. _`Compose file reference`: https://docs.docker.com/compose/compose-file/
+
+Database Migration
+``````````````````
+
+Follow instructions to `Upgrade from InfluxDB 1.x to 2.7 with Docker`_.
+
+.. _`Upgrade from InfluxDB 1.x to 2.7 with Docker`:
+https://docs.influxdata.com/influxdb/v2/install/upgrade/v1-to-v2/docker/
+
+Refer to Confluence for specific instructions on migrating for SO labs.
+
+Grafana
```

Comment:
Does this configuration change at all with the new auth token, organization, and bucket information in v2?